packages/veritone-react-common/src/components/Video/MediaPlayer/styles.scss

```diff
@@ -0,0 +1,45 @@
+.mediaPlayer {
```

Comment:
any reason we're nesting class names here? if not then unnest them to keep them as flat as possible please